<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>runner</class>
 <widget class="QMainWindow" name="runner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Runner</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="openButton">
       <property name="text">
        <string>Otwórz</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="placeholderText">
        <string>Otwórz plik samp.exe</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>224</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(85, 85, 255);
font-size: 14px;
font-family: Arial;
font-weight: bold;</string>
       </property>
       <property name="text">
        <string>Najnowsze tematy na Pawno.pl</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="linkButton_1">
       <property name="styleSheet">
        <string notr="true">#linkButton_1 {
	border: none;
}</string>
       </property>
       <property name="text">
        <string>Loading...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="link_1">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>_____________________________________</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="linkButton_2">
       <property name="styleSheet">
        <string notr="true">#linkButton_2 {
	border: none;
}</string>
       </property>
       <property name="text">
        <string>Loading...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="link_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>_____________________________________</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="linkButton_3">
       <property name="styleSheet">
        <string notr="true">#linkButton_3 {
	border: none;
}</string>
       </property>
       <property name="text">
        <string>Loading...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="link_3">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>_____________________________________</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="linkButton_4">
       <property name="styleSheet">
        <string notr="true">#linkButton_4 {
	border: none;
}</string>
       </property>
       <property name="text">
        <string>Loading...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="link_4">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>_____________________________________</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="linkButton_5">
       <property name="styleSheet">
        <string notr="true">#linkButton_5 {
	border: none;
}</string>
       </property>
       <property name="text">
        <string>Loading...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="link_5">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>_____________________________________</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>START!</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="ipEdit">
       <property name="placeholderText">
        <string>Ustaw IP serwera</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="portEdit">
       <property name="placeholderText">
        <string>Ustaw port serwera</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
